next-tavla/app/(admin)/components/CreateBoard/index.tsx

```diff
@@ -155,25 +168,111 @@ function NameAndOrganizationSelector({
-            >
-                Jeg vil ikke velge organisasjon
-            </Checkbox>
+            {!createorg && (
+                <>
+                    <Heading4>Legg til i en organisasjon</Heading4>
+                    <Label>Velg en organisasjon</Label>
+                    <Dropdown
+                        items={organizations}
+                        label="Dine organisasjoner"
+                        selectedItem={selectedOrganization}
+                        onChange={setSelectedOrganization}
+                        clearable
+                        className="mb-4"
+                        aria-required="true"
+                        disabled={personal}
+                        {...getFormFeedbackForField('organization', state)}
+                    />
+                    {!selectedOrganization && (
+                        <div className="flex flex-row">
+                            <IconButton
+                                onClick={() => setCreateOrg(true)}
+                                className="gap-2"
+                            >
+                                <AddIcon /> Opprett ny organisasjon
+                            </IconButton>
+                            <Checkbox
+                                checked={personal}
+                                onChange={() => setPersonal(!personal)}
+                                name="personal"
+                            >
+                                Jeg vil ikke velge organisasjon
+                            </Checkbox>
+                        </div>
+                    )}
+                </>
+            )}
+            <div>
+                {createorg && (
+                    <div
+                        className="w-full"
+                        aria-live="polite"
+                        aria-relevant="all"
+                    >
+                        <Label className="weight500">
+                            Opprett ny organisasjonen
+                        </Label>
+                        <TextField
+                            size="medium"
+                            label="Organisasjonsnavn"
+                            required
+                            aria-required
+                            ref={orgName}
+                            {...getFormFeedbackForField('organization', state)}
+                        />
+                        <FormError
+                            {...getFormFeedbackForField('general', state)}
+                        />
+                        <div className="mt-2">
+                            <Button
+                                variant="secondary"
+                                onClick={() => setCreateOrg(false)}
+                            >
+                                Avbryt
+                            </Button>
+                            <SubmitButton
+                                variant="secondary"
+                                className="ml-6"
```

Comment:
Dette kan også være personlig preferanse, men jeg er mer fan av å unngå å sette margins osv på spesifikke komponenter når man skal distansere de fra noe annet. Da vil det se rart ut om de plutselig blir liggende på hver sin "rad", slik som dette: 
![image](https://github.com/entur/tavla/assets/43408175/3a6a8f22-5e9d-47cd-b1d1-afe695dbd00a)

Et forslag kan være at man legger på `flex gap-6` på den diven som wrapper disse to knappene (linje 226) - da kan ma gjøre den mobilvennlig med det samme 😄